examples/wallet/app/containers/SendTransaction.js

```diff
@@ -3,13 +3,23 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import SendTransaction from '../components/SendTransaction';
 import { sendTransaction } from '../actions/account';
+import { buildSendFundsTransaction } from '../utils/wasmWrapper';
+import { broadcastTransaction } from '../utils/nodeConnection';
 
 function mapStateToProps(state) {
   return { balance: state.account.balance, nodeSettings: state.nodeSettings };
 }
 
 function mapDispatchToProps(dispatch) {
-  return bindActionCreators({ sendTransaction }, dispatch);
+  return bindActionCreators(
+    {
+      sendTransaction: sendTransaction(
```

Comment:
Did you consider using a slightly different name for the `sendTransaction` function? Although I'm not sure if it would be better, but something like `buildSendTransactionAction`. Anyways, it's not something that confusing so the verbosity may be unwarranted

Reply:
I think it might make sense to change it, more so considering how the exported type is valid for the containers but not for the components :upside_down_face: